charts/victoria-metrics-k8s-stack/README.md

```diff
@@ -282,6 +282,14 @@ See the history of versions of `vmks` application with command.
 helm history vmks -n NAMESPACE
 ```
 
+### Install operator separately
+
+In order to have more control over operator and managed resources it's recommended to disable operator in k8s-stack chart (victoria-metrics-operator.enabled: false) and [install it](https://docs.victoriametrics.com/helm/victoriametrics-operator/) separately. To move operator from existing k8s-stack release please follow the steps below:
```

Comment:
>In order to have more control over operator and managed resources

What additional control do users gain by installing the operator separately, make uninstall more flexible?

Could you give more detailed setup guide, including install operator via vm-operator chart first and verify it's working maybe?

Reply:
pre-delete hooks are not supported in ArgoCD and it's not working as expected in FluxCD < 2.4.0.